stable/vault/values.yaml

```diff
@@ -4,7 +4,7 @@
 replicaCount: 1
 image:
   repository: vault
-  tag: 0.9.1
```

Comment:
Again a minor:) You changed this to 0.9.3 but in the stable/vault/Chart.yaml still contains 0.9.1

Reply:
This has been fixed now please verify.